cache/search_stats.go

```diff
@@ -190,15 +192,25 @@ func (ssc *SearchStatsCacheImpl) IsTagWithUnits(uid string, cts ...string) bool
 }
 
 func (ssc *SearchStatsCacheImpl) IsTagWithEnoughUnits(uid string, count int, cts ...string) bool {
-	return ssc.isClassWithUnits("tags", uid, count, cts...)
+	return ssc.isClassWithUnits("tags", uid, &count, nil, cts...)
 }
 
 func (ssc *SearchStatsCacheImpl) IsSourceWithUnits(uid string, cts ...string) bool {
 	return ssc.IsSourceWithEnoughUnits(uid, 1, cts...)
 }
 
 func (ssc *SearchStatsCacheImpl) IsSourceWithEnoughUnits(uid string, count int, cts ...string) bool {
-	return ssc.isClassWithUnits("sources", uid, count, cts...)
+	return ssc.isClassWithUnits("sources", uid, &count, nil, cts...)
+}
+
+func (ssc *SearchStatsCacheImpl) IsTagHasSingleUnit(uid string, cts ...string) bool {
+	one := 1
```

Comment:
The one parameter is strange why you have to assign it like this?

Reply:
the get the pointer address of value 1